docs/samples/logger/local/README.md

```diff
@@ -0,0 +1,109 @@
+# Local Test of Kfserving Logger
+
+In one terminal start an echo http server on port 8000
+
+```
+docker run -it -p 8000:80 --rm -t mendhak/http-https-echo
+```
+
+Start an SKLearn Iris model on port 8081
+
+```
+python -m sklearnserver --model_dir gs://kfserving-samples/models/sklearn/iris --model_name sklearn-iris --http_port 8081
+```
+
+Start the Kfserving logger from Kfserving root folder:
+
+```
+
+bin/logger --log_url http://0.0.0.0:8000 --svc_port 8081 --log_mode all
+```
+
+Send a request:
+
+```
+curl -v -d @./input.json http://0.0.0.0:8080/v1/models/sklearn-iris:predict -H "Content-Type: application/json"
```

Comment:
Is the header necessary? I never need it when I curl.

Reply:
I've removed that. Though our payloads are all JSON so not sure its valid we don't require that content-type in header otherwise will default to form-encoded.